server/store.go

```diff
@@ -378,3 +385,34 @@ func (p *Plugin) getLastReminderTimeForUser(userID string) (int64, error) {
 
 	return reminderAt, nil
 }
+
+func (p *Plugin) saveReminderPreference(userID string, preference bool) error {
+	preferenceString := strconv.FormatBool(preference)
+	appErr := p.API.KVSet(reminderEnabledKey(userID), []byte(preferenceString))
+	if appErr != nil {
+		return errors.New(appErr.Error())
```

Comment:
Cant this be 
```suggestion
		return appErr
```
instead?

Reply:
simplified as suggested